src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,264 @@
+    df_non_diploid = read_non_diploid(filename_cnv_qvalues)
+    df_manta_high_outlier = read_outlier(filename_high_manta, get_col_name("manta", "high"))
+    df_melt_high_outlier = read_outlier(filename_high_melt, get_col_name("melt", "high"))
+    df_wham_high_outlier = read_outlier(filename_high_wham, get_col_name("wham", "high"), 11)
+    df_total_high_outliers = read_all_outlier(filename_high_manta, filename_high_melt, filename_high_wham, "high")
+    df_manta_low_outlier = read_outlier(filename_low_manta, get_col_name("manta", "low"))
+    df_melt_low_outlier = read_outlier(filename_low_melt, get_col_name("melt", "low"))
+    df_wham_low_outlier = read_outlier(filename_low_wham, get_col_name("wham", "low"), 11)
+    df_total_low_outliers = read_all_outlier(filename_low_manta, filename_low_melt, filename_low_wham, "low")
+
+    # all data frames
+    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
+           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+           df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers]
+    output_df = reduce(
+        lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs
+    ).fillna(0)  # "none"
+
+    # save the file
+    return output_df.to_csv(f"{output_prefix}evidence_qc_table.tsv", sep="\t", header=True, index=False)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "ploidy_test_filename",
+        help="Sets the filename containing ploidy tests.")
+
+    parser.add_argument(
+        "median_cov_transposed_filename",
+        help="Sets the filename containing median coverage.")
+
+    parser.add_argument(
+        "wgd_scores_filename",
+        help="Sets the filename containing WGD scores.")
+
+    parser.add_argument(
+        "binwise_cnv_qValues_filename",
+        help="Sets the filename containing bin-wise CNV q-values.")
+
+    parser.add_argument(
+        "manta_qc_outlier_high_filename",
+        help="Sets the filename containing Manta QC outlier high.")
+
+    parser.add_argument(
+        "melt_qc_outlier_high_filename",
+        help="Sets the filename containing Melt QC outlier high.")
+
+    parser.add_argument(
+        "wham_qc_outlier_high_filename",
+        help="Sets the filename containing Wham QC outlier high.")
+
+    parser.add_argument(
+        "manta_qc_outlier_low_filename",
+        help="Sets the filename containing Manta QC outlier low.")
+
+    parser.add_argument(
+        "melt_qc_outlier_low_filename",
+        help="Sets the filename containing Melt QC outlier low.")
+
+    parser.add_argument(
+        "wham_qc_outlier_low_filename",
+        help="Sets the filename containing Wham QC outlier low.")
+
+    parser.add_argument(
+        "melt_insert_size_file",
+        help="Sets the filename containing Melt insert size.")
+
+    parser.add_argument(
+        "--output-prefix",
+        default="",
```

Comment:
```suggestion
```

Reply:
@epiercehoffman I'm not sure if I understand the suggestion here. Can you please explain the suggestion, here?

Reply:
I think the prefix should be a required argument, so it does not need a default value